applications/TreasuryTracker.md

```diff
@@ -0,0 +1,180 @@
+# TreasuryTracker
+
+- **Team Name:** Stake Plus Inc
+- **Payment Address:** 0x06737F4137E244DBc46c6044E5051A66074F3f87
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview :page_facing_up:
+
+### Problem Statement
+
+The community is plunging into collectives without any established budgeting mechanisms. This absence of financial oversight is compounded by misunderstandings and a general lack of clarity concerning the specifics of different operations within the ecosystem. Without a clear grasp of current spending on categorized requests, it's impossible to set up appropriate budgets, resulting in a lack of control and potential misallocation of resources.
+
+### Overview
+
+TreasuryTracker is an innovative analytics portal designed to provide comprehensive insights into the OpenGovernance referenda on Polkadot and Kusama. Through intuitive charts, tables, and reports, the platform aims to track, analyze, and present vital data, allowing the community to understand areas of treasury funding and obtain a holistic overview of the governance process.
+
+Our interest in creating this project comes from a shared vision of providing the community with the tools needed to actively engage in the governance process, ensure efficient utilization of treasury funds, and create a more accountable and transparent ecosystem.
+
+
+### Project Details
+
+#### Mockups/Designs of UI Components
+
+Mockups have been created to display essential features, including interactive charts, tables, and reporting tools. The main dashboard will include an intuitive navigation menu and provide visualizations of various metrics related to the OpenGovernance referenda on Polkadot and Kusama. 
+
+Within the following grant, we plan to implement following two analytics:
+
+- A Weekly Cumulative Overview: Tracking New Referenda by Project Categories
+
+![1](https://raw.githubusercontent.com/stake-plus/Grants-Program/master/assets/pic%201.png)
```

Comment:
How do you create these categories? Do they already exist (i.e. are they encoded on-chain) or are they somehow estimated?

Reply:
The base categories are the ones that the web3 foundation uses internally for tracking of these proposals. You can see the full sheet implementation of the charting at the URL included:

- Infrastructure
- Network Security
- Ecosystem
- Marketing
- Community
- Software
- Bounties
- Others

We intend to manually tag proposals with sub-tags (TBD) that reference each one of these main tags so that users would be able to view specific categories and sub categories. We are happy to work with the W3F on a more formal standard for this categorization. 

https://docs.google.com/spreadsheets/d/1n0DImi-8qLAEx3lC9lNz_tJjpUrmD7YqofLYGLkrN9o/edit?usp=sharing